beeflow/common/crt/charliecloud_driver.py

```diff
@@ -101,9 +101,14 @@ def run_text(self, task):  # noqa
             container_path = '/'.join([self.container_archive, task_container_name]) + '.tar.gz'
```

Comment:
```suggestion
            container_path = '/'.join([self.container_archive, task_container_name]) + '.tar.gz'
```
Is this line needed anymore?